ecl/hql/hqlgram2.cpp

```diff
@@ -1013,7 +1013,11 @@ IHqlExpression * HqlGram::processEmbedBody(const attribute & errpos, IHqlExpress
         }
     }
     else
```

Comment:
This test condition is wrong.  Embed, activity
- does not support transforms, rows
- does support actions.